aio/deploy/helm-chart/kubernetes-dashboard/values.yaml

```diff
@@ -0,0 +1,176 @@
+# Default values for kubernetes-dashboard
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+# name: value
+
+image:
+  repository: kubernetesui/dashboard
+  tag: v2.0.0-rc1
+  pullPolicy: IfNotPresent
+  pullSecrets: []
+
+replicaCount: 1
+
+## Here annotations can be added to the kubernetes dashboard deployment
+annotations: {}
+## Here labels can be added to the kubernetes dashboard deployment
+labels: {}
+
+## Serve application over HTTP without TLS
+##
+## Note: If set to true, you may want to add --enable-insecure-login to extraArgs
+protocolHttp: false
+
+## Additional container arguments
+##
+# extraArgs:
+#   - --enable-skip-login
+#   - --enable-insecure-login
+#   - --system-banner="Welcome to Kubernetes"
+
+## Additional container environment variables
+##
+extraEnv: []
+# - name: SOME_VAR
+#   value: 'some value'
+
+# Annotations to be added to kubernetes dashboard pods
+podAnnotations:
+  seccomp.security.alpha.kubernetes.io/pod: 'runtime/default'
+
+## Node labels for pod assignment
+## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+##
+nodeSelector: {}
+
+## List of node taints to tolerate (requires Kubernetes >= 1.6)
+tolerations: []
+#  - key: "key"
+#    operator: "Equal|Exists"
+#    value: "value"
+#    effect: "NoSchedule|PreferNoSchedule|NoExecute"
+
+## Affinity
+## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+affinity: {}
+
+# priorityClassName: ""
+
+service:
+  type: ClusterIP
+  externalPort: 443
+
+  # LoadBalancerSourcesRange is a list of allowed CIDR values, which are combined with ServicePort to
+  # set allowed inbound rules on the security group assigned to the master load balancer
+  # loadBalancerSourceRanges: []
+
+  ## Additional Kubernetes Dashboard Service annotations
+  annotations: {}
+
+  ## Here labels can be added to the Kubernetes Dashboard service
+  labels: {}
+
+resources:
+  requests:
+    cpu: 100m
+    memory: 100Mi
+  limits:
+    cpu: 10
```

Comment:
Should this really be set to `10` cpu core limit by default? Or is this perhaps a typo? Noticed my 2-core t3.medium was way committed, and it boiled down to this pod having a limit of 10. Seems terribly high.

Reply:
This is a typo, thank you very much.
I would suggest a default of 2 cpus (more than 1, in order not to be throttled by kernel to 1.00 and sometimes a bit less due to approximations of CFS.

Note that Kubernetes good practices tend to go to NO limits.cpu. I tend to be on the middle : set the limit high enough so that it is not a problem, but still save the VM if a pod goes rogue.

Reply:
2 sounds reasonable to me, and I agree NO limits.cpu is not likely to be the best course of action for most Pods including this one.